src/routes/userRoutes.js

```diff
@@ -0,0 +1,69 @@
+const { Router } = require("express");
+const express = require("express");
+const { body, validationResult } = require("express-validator");
+const User = require("../models/User");
+const bcrypt = require("bcryptjs");
+
+const router = Router();
+
+router.use(express.json());
+//new user registration
+router.post(
+  "/user_create",
```

Comment:
according to REST API only one route should be used for CRUD. this one is `/user`. 

the difference should be in methods only.
for example,
the route is: `/user`
methods:
1. **POST** - creates a new user
2. **PATCH** - updates one
3. **GET** - retrieves the list of ones
4. **DELETE** - removes one

Reply:
Done.